<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;S-Core Dream&quot;;
background-color:white;</string>
  </property>
  <widget class="QLineEdit" name="login_pwd">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>290</y>
     <width>261</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid rgb(182, 18, 22);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="login_ok">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>350</y>
     <width>390</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(182, 18, 22);
color:white;
font: 16pt &quot;S-Core Dream&quot;;</string>
   </property>
   <property name="text">
    <string>로그인</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>ID(EMAIL)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_email">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>240</y>
     <width>261</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid rgb(182, 18, 22);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>231</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(182, 18, 22);
font: 27pt &quot;a타임머신&quot;;</string>
   </property>
   <property name="text">
    <string>날씨우체통</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_back">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt  &quot;S-Core Dream&quot;;
background-color:rgb(248, 248, 248)</string>
   </property>
   <property name="text">
    <string>뒤로가기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
